$OpenBSD: patch-source_fitz_load-jpx_c,v 1.4 2017/05/05 14:02:51 sthen Exp $

If OPJ_STATIC is defined, load-jpx.c linkage with libopenjp2.so is broken
("relocation R_X86_64_PC32 against symbol ... can not be used when making a
shared object") 

Index: source/fitz/load-jpx.c
--- source/fitz/load-jpx.c.orig
+++ source/fitz/load-jpx.c
@@ -444,7 +444,6 @@ fz_load_jpx_info(fz_context *ctx, unsigned char *data,
 
 #else /* HAVE_LURATECH */
 
-#define OPJ_STATIC
 #define OPJ_HAVE_INTTYPES_H
 #if !defined(_WIN32) && !defined(_WIN64)
 #define OPJ_HAVE_STDINT_H
